react/ui-components/src/molecules/stories/FormCard.stories.js

```
@@ -429,4 +424,187 @@ export const WithLayout = () => {
+    setSelectedGuardian(option.code);
+  };
+
+  const handleCheckboxChange = (event) => {
+    setIsSameAsPropertyAddress(event.target.checked);
+  };
+
+  const textBlockStyle = {
+    width: isMobileView ? "100%" : "30%",
+  };
+
+  const headerData = [<TextBlock header={"Enter Details"}></TextBlock>];
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)